test/helpers/db.js

```diff
@@ -0,0 +1,59 @@
+const Schema = require('@asl/schema');
+
+const settings = {
+  database: process.env.DATABASE_NAME || 'asl-test',
+  user: process.env.DATABASE_USERNAME || 'postgres',
+  host: process.env.DATABASE_HOST || 'localhost',
+  password: process.env.DATABASE_PASSWORD || 'test-password'
+};
+
+module.exports = () => {
+
+  return {
+    init: (populate) => {
+      const schema = Schema(settings);
+      const tables = [
+        'Notification',
+        'TrainingPil',
+        'TrainingCourse',
+        'EmailPreferences',
+        'AsruEstablishment',
+        'ProjectEstablishment',
+        'ProjectProfile',
+        'RetrospectiveAssessment',
+        'Procedure',
+        'Rop',
+        'ProjectVersion',
+        'Project',
+        'Permission',
+        'Authorisation',
+        'FeeWaiver',
+        'PilTransfer',
+        'PIL',
+        'PlaceRole',
+        'Place',
+        'Role',
+        'Exemption',
+        'Certificate',
+        'Profile',
+        'Invitation',
+        'Establishment'
+      ];
```

Comment:
(optional) it's a bit annoying having to maintain these table lists when we add new tables - maybe we should start grabbing them from the db?
```SELECT tablename FROM pg_tables WHERE schemaname = 'public' AND tablename NOT LIKE 'knex%';```

Reply:
They need to be deleted in a specific order to deal with the foreign, so that likely wouldn't work. I'll give something similar a whirl though and see.

Reply:
Rather than calling `.delete()` which is blocked by foreign keys, maybe try:
```TRUNCATE TABLE <tableName> CASCADE;``` which should just delete any rows that link to that table